src/components/PageHeadingFlowRuns.vue

```diff
@@ -1,14 +1,24 @@
 <template>
   <page-heading class="page-heading-flow-runs" :crumbs="crumbs">
-    <template #actions>
+    <template v-if="!hideActions" #actions>
       <SavedFilters />
     </template>
   </page-heading>
 </template>
 
 <script lang="ts" setup>
+  import { defineProps, withDefaults, computed } from 'vue'
   import PageHeading from '@/components/PageHeading.vue'
   import  SavedFilters from '@/components/SavedFilters.vue'
 
+
+  const props = withDefaults(defineProps<{
+    hideActions?: boolean,
+  }>(), {
+    hideActions: false,
+  })
+
+  const hideActions = computed(() => props.hideActions)
```

Comment:
I'm guessing this line is only here to ensure props.hideActions is reactive? I don't think that's necessary here (@pleek91 can you verify?), but if it is I would suggest vue's `toRef`

```ts
const hideActions = toRef(props, 'hideActions')
```

Reply:
If you're only using it in the template it's not needed.

Reply:
@nicolascribbles I still don't think you need this line

Reply:
@stackoverfloweth updated!